drivers/sx127x/sx127x_netdev.c

```diff
@@ -343,6 +342,16 @@ static int _get(netdev_t *netdev, netopt_t opt, void *val, size_t max_len)
             *((netopt_enable_t*) val) = sx127x_get_rx_single(dev) ? NETOPT_ENABLE : NETOPT_DISABLE;
             break;
 
+        case NETOPT_TX_POWER:
+            assert(max_len >= sizeof(int8_t));
```

Comment:
Why do we used assert in every case of this switch ? What's the point ? We could just do it once before the switch call

Reply:
@miri64  can answer this better than me

Reply:
@dylad the width of the value depends on the option used (see doc in `netopt.h` for details), so it needs to be checked per option.

Reply:
(with width I mean the type and its width in bytes if that's not clear)

Reply:
Ok I understand now then this is fine. (One day I'll really take a deep look at this black magic).